test/blockgaslimit.js

```diff
@@ -0,0 +1,119 @@
+const Web3 = require("web3");
+const assert = require("assert");
+const Ganache = require("../index");
+const path = require("path");
+const compileAndDeploy = require("./helpers/contracts").compileAndDeploy;
+
+const TARGET = process.env.TARGET || "ganache";
+const PARITY_UNLOCKED_ADDR = process.env.PARITY_UNLOCKED_ADDR || "0x00a329c0648769A73afAc7F9381E08FB43dBEA72";
+const mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
+
+function setUp(options = { mnemonic }, contractName = "BlockGasLimit") {
+  const context = {
+    options: options,
+    provider: null,
+    web3: null,
+    accounts: [],
+    contractArtifact: {},
+    instance: null
+  };
+
+  before("setup web3", async function() {
+    // Enable Ganache provider
+    if (TARGET === "ganache") {
+      // eslint-disable-next-line new-cap
+      context.provider = new Ganache.provider(context.options);
```

Comment:
just use `Ganache.provider(context.options)` without the `new` keyword here and remove the eslint comment.